src/Emulator.cpp

```diff
@@ -15,48 +15,74 @@
  *
  * @section DESCRIPTION
  *
- * Chip8 emulator that uses SDL
+ * Creates the windows and handles different events
  *
  */
 
 #include <iostream>
 
-#include <SDL2/SDL.h>
+#include "Emulator.h"
 
-int main(int argc, char* args[])
+Emulator::Emulator()
```

Comment:
Is this how you declare an object?

Reply:
No, this is actually the constructor. I created the Emulator object in line 7 of Runner.cpp.

Reply:
It looks like the Emulator.cpp was before the main.cpp. What kind of program Emulator.cpp is suppose to run?